frontend/src/styles/components/EmailFooter.scss

```diff
@@ -0,0 +1,104 @@
+$email-icon--width: 146px;
+$email-icon--height: 127px;
+
+$email-icon--responsive-unit: .66;
+
+.email-footer {
+  background: $transparent-white-1;
```

Comment:
@johngruen had similar issues that were solved by blowing away some old `node_modules`. This variable [exists and is used all over](https://github.com/mozilla/testpilot/search?utf8=%E2%9C%93&q=transparent-white-1).